docs/configure/upgrading.md

```diff
@@ -2,44 +2,44 @@
 
 ## Upgrade script
 
-The most common upgrade is Storybook itself. Storybook follows [Semantic Versioning](https://semver.org/). We release patch releases with bugfixes continuously, minor versions of Storybook with new features every few months, and major versions of Storybook with breaking changes roughly once per year.
+The most common upgrade is Storybook itself. [Storybook releases](/releases) follow [Semantic Versioning](https://semver.org/). We publish patch releases with bugfixes continuously, minor versions of Storybook with new features every few months, and major versions of Storybook with breaking changes roughly once per year.
 
 To help ease the pain of keeping Storybook up-to-date, we provide a command-line script:
 
 ```sh
 npx sb@next upgrade
 ```
 
-This upgrades all your Storybook packages to the latest stable version, perform sanity checks of your package versions, and also check for [automigrations](#automigrate) to automatically update your configuration.
+This upgrades all of the Storybook packages in your project to the latest stable version, performs confidence checks of your package versions, and checks for opportunities to run [automigrations](#automigrate) to automatically update your configuration.
 
 <div class="aside">
 
-In addition to running the command, we also recommend skimming [MIGRATION.md](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md), an exhaustive log of changes relevant changes and deprecations that might affect your upgrade.
+In addition to running the command, we also recommend skimming [MIGRATION.md](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md), an exhaustive log of relevant changes and deprecations that might affect your upgrade.
 
 </div>
 
 ## Automigrate script
 
-Storybook upgrades are not the only thing that can break your Storybook: changes in the ecosystem are also present challenges. For example, lots of frameworks (Angular 12, Create React App v5, NextJS) have recently migrated from webpack4 to webpack5, so even if you don't upgrade your Storybook version, you might need to update your configuration accordingly. That's what Automigrate is for:
+Storybook upgrades are not the only thing to consider: changes in the ecosystem also present challenges. For example, lots of frameworks ([Angular 12](https://angular.io/guide/updating-to-version-12#breaking-changes-in-angular-version-12), [Create React App v5](https://github.com/facebook/create-react-app/pull/11201), [NextJS](https://nextjs.org/docs/upgrading#webpack-5)) have recently migrated from [webpack 4 to webpack 5](https://webpack.js.org/migrate/5/), so even if you don't upgrade your Storybook version, you might need to update your configuration accordingly. That's what Automigrate is for:
```

Comment:
I attempted to add links to the most helpful spot for each of these. Not sure if that's actually more helpful than just linking to the projects' homepages. 🤷‍♂️

Reply:
It's great. 💯